sequitur.go

```diff
@@ -279,7 +286,14 @@ func rawPrint(w io.Writer, r *rules) error {
 				return err
 			}
 		} else {
-			if _, err := w.Write([]byte{byte(p.value)}); err != nil {
+			rb := make([]byte, utf8.UTFMax)
+			sz := 1
+			if p.value <= 0xff {
```

Comment:
This doesn't look right to me - it should be < 0x80 (utf8.RuneSelf). I think I'd just use EncodeRune regardless; the overhead is small.

Reply:
I was getting errors when printing the binary value when using EncodeRune for values in the range utf.RuneSelf < r < 0xff, hence the change